### DISCUSSION TITLE
Is this repo only for CS oriented jobs?

### DISCUSSION THREAD

**Discussion by kanjurer** [jenndryden/Canadian-Tech-Internships-and-New-Grad-2025#160] [Q&A]
Are we only posting jobs desirable for a cs major, or can we also post positions for engineering students (CE). So far I have only been seeing software development internships lol

**Comment by jenndryden** [comment]
So far we have been focused on SWE (software) related roles but I think it would be interesting to also include computer related internships! Open to other thoughts as well 😊

**Comment by ypatel2022** [comment]
I would love that! (im in CE)

**Comment by rohan057** [comment]
Hi guys we have a Graduate Programme for 2024 if interested read below and apply.

Graduate Programme 2024: Software Development

About Us:
Accolite Digital, a leading digital transformation services provider, is excited to announce the launch of its Software Development Graduate Programme in 2024.

Position: Software Developer - Python/Java

Location: Glasgow

Programme Start: January 2024

Your Role:
As a Software Developer with Accolite Digital, you will have the opportunity to work on-site, starting in January 2024. Your initial engagement will be with a Fortune 100 global investment bank in the financial services industry. Responsibilities include designing, developing, modifying, and maintaining software products.

What We Offer:

Comprehensive development plan
Extensive and continuous training in banking, financial services, and relevant technologies
Exposure to cutting-edge technology and a dynamic community of technologists
Requirements:

MSc/BSc in a technical-related degree (Computer Science, Computing-Related Fields)
Graduated between 2019-2022
Genuine interest in technology and programming
Coding proficiency in Java and/or Python
Strong understanding of object-oriented concepts
Attention to detail, quality-focused, and adherence to standards/best practices
Ability to thrive in a fast-paced environment
Excellent written and verbal communication skills
About Accolite Digital:
Accolite Digital is an innovative digital transformation services provider, applying design thinking to solve complex business problems for Fortune 500 companies. We believe in a bias-free world and encourage diversity in our workforce.

Why Join Us:
At Accolite Digital, we believe a bias-free world is an empowered world. We welcome applications from everyone, especially women and individuals from diverse backgrounds, to contribute to building the next generation of Accolite Digital.

Ready to Launch Your Career in Software Development?
Apply now for the Software Development Graduate Programme and be part of a community of bright and passionate technologists shaping the future at Accolite Digital.

Python Programme: https://recruiterflow.com/db_74f856c0d97ab5b8dfdec4834ea8901c/jobs/37
Java Programme: https://recruiterflow.com/db_74f856c0d97ab5b8dfdec4834ea8901c/jobs/37

Reply back if you have any further questions and I will be happy to answer, else best of luck with your applications.

**Reply by kanjurer** [reply]
Hey, I do not believe this repo is for advertisement, and is only for internships in Canada (seems like the position you posted in based out of UK)

(Correct me if I am wrong @jenndryden or if we want to have a separate forum for job advertisements, that would be nice too)

**Reply by jenndryden** [reply]
Hey @kanjurer! Yes, I agree with everything you are saying. Also agree that a separate forum for job advertisements could be interesting, especially if there are any startups or companies looking for freelance interns!

@rohan057 unfortunately this is a repo for Canadian internships but we appreciate you reaching out 😊
